.gitlab/package_build/deb.yml

```
@@ -304,6 +304,22 @@ dogstatsd_deb-arm64:
     paths:
       - $OMNIBUS_PACKAGE_DIR
 
+updater_deb-amd64:
+  extends: .updater_build_common_deb
+  rules:
+    !reference [.on_a7]
+  stage: package_build
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["arch:amd64"]
+  needs: ["go_mod_tidy_check", "go_deps", "generate_minimized_btfs_amd64"]
+  variables:
+    AGENT_MAJOR_VERSION: 7
+    PACKAGE_ARCH: arm64
```

Shouldn't be this instead ?
```suggestion
    PACKAGE_ARCH: amd64
```